.pre-commit-config.yaml

```diff
@@ -0,0 +1,49 @@
+ci:
+  autofix_prs: false
+  autoupdate_schedule: quarterly
+
+repos:
+
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.6.0
+  hooks:
+  - id: trailing-whitespace
+    exclude: pheps/phep-0001/workflow.dot
+  - id: check-case-conflict
+  - id: end-of-file-fixer
+    exclude: pheps/.*/.*\.dot
+  - id: check-yaml
+  - id: check-merge-conflict
+
+- repo: https://github.com/sirosen/texthooks
+  rev: 0.6.6
+  hooks:
+  - id: fix-smartquotes
+  - id: fix-spaces
+  - id: fix-ligatures
+  - id: forbid-bidi-controls
+
+- repo: https://github.com/pre-commit/pygrep-hooks
+  rev: v1.10.0
+  hooks:
+  - id: text-unicode-replacement-char
+
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.13.0
+  hooks:
+  - id: pretty-format-yaml
+    args: [--autofix]
+
+- repo: https://github.com/python-jsonschema/check-jsonschema
+  rev: 0.29.0
+  hooks:
+  - id: check-github-workflows
+
+- repo: https://github.com/codespell-project/codespell
+  rev: v2.3.0
+  hooks:
+  - id: codespell
+    args: [--write-changes]
+    exclude: .*\.dot|.*\.svg
```

Comment:
This might be worth as a global exclude?

Reply:
Good call!